global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "5.0.100-rc.1.20379.10"
+    "version": "5.0.100-preview.7.20330.3"
```

Comment:
Suggest we use the public preview version, `5.0.100-preview.7.20366.6`. Hopefully this is a very short-lived change, so this is a nit.